Data/collections.json

```diff
@@ -6198,7 +6198,7 @@
                     "rel": "about"
                 },
                 {
-                    "href": "https://sentinelshare.page.link/918j",
+                    "href": "link.dataspace.copernicus.eu/xvs",
```

Comment:
I think you should add the full link with https:

Reply:
Done.